docs/_config.yml

```diff
@@ -26,7 +26,8 @@ url_root: http://0.0.0.0:4000
 path:
   style: /style
   guide: /guide
-  website: /website
+  #guide: "/v/latest"
```

Comment:
aha i see why you did what you did -- but it needs to be reverted.

we like to be able to run `jekyll serve` from the root and for the resulting links to be happy.  this means we have to have guide in guide and website it website.  this is why the structural changes are done afterwards production builds.  unfortunate but i don't see a way around it.